src/ebmc/ebmc_properties.h

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************\
+
+Module: Data Structure for the Properties
+
+Author: Daniel Kroening, dkr@amazon.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_EBMC_PROPERTIES_H
+#define CPROVER_EBMC_PROPERTIES_H
+
+#include <util/cmdline.h>
+#include <util/message.h>
+
+#include <solvers/prop/literal.h>
+#include <trans-netlist/trans_trace.h>
+
+#include "transition_system.h"
+
+class ebmc_propertiest
+{
+public:
+  struct propertyt
+  {
+  public:
+    unsigned number;
```

Comment:
Is `unsigned` the right type here, or should it perhaps be `std::size_t`?

Reply:
Also, add `= 0` to simplify the default constructor.

Reply:
will change to `size_t`.